modules/transport.js

```diff
@@ -823,6 +824,7 @@ Transport.prototype.internal = {
 				});
 			}
 
+			peer.state = Peer.STATE.ACTIVE;
```

Comment:
Assuming because of this change summary: `assign state active after proper response`.